src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,10 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    public static Ball.Colors getRandomColor() {
+        int index = new Random().nextInt(Ball.Colors.values().length);
+        return Ball.Colors.values()[index];
     }
```

Comment:
The method getRandomColor is tightly coupled with the enum inside the Ball class. Consider decoupling it by passing the array of colors as a parameter or by using a more generic approach.